miden-tx/src/host/mod.rs

```diff
@@ -122,11 +125,13 @@ impl<A: AdviceProvider, T: TransactionAuthenticator> TransactionHost<A, T> {
         Ok(())
     }
 
-    fn on_note_add_asset<S: ProcessState>(
+    /// Adds an asset from the stack to the note stored by its pointer.
```

Comment:
nit: I would write this as: "Adds an asset at the top of the [OutputNoteBuilder] identified by the note pointer."